crates/cdk-mintd/src/main.rs

```diff
@@ -359,12 +361,18 @@ async fn main() -> anyhow::Result<()> {
     }
 
     let shutdown = Arc::new(Notify::new());
-
+    let mint_clone = Arc::clone(&mint);
     tokio::spawn({
         let shutdown = Arc::clone(&shutdown);
-        async move { mint.wait_for_paid_invoices(shutdown).await }
+        async move { mint_clone.wait_for_paid_invoices(shutdown).await }
     });
 
+    let _rpc_shutdown = Arc::new(Notify::new());
+
+    let mint_rpc = MintRPCServer::new("127.0.0.1", 8086, mint)?;
```

Comment:
Would it make sense to place it behind a feature flag, like the swagger UI?

Reply:
resolved in 4ce1337a9397a6774b39a43fa779ee9d4bd8b3c2